common-service-core/src/main/java/org/zowe/apiml/security/HttpsFactory.java

```diff
@@ -67,7 +67,7 @@ public CloseableHttpClient createSecureHttpClient(HttpClientConnectionManager co
     }
 
     public ConnectionSocketFactory createSslSocketFactory() {
-        if (config.isVerifySslCertificatesOfServices() || config.isNonStrictVerifySslCertificatesOfServices()) {
+        if (config.isVerifySslCertificatesOfServices()) {
```

Comment:
Shouldn't 
```
config.isVerifySslCertificatesOfServices() && !config.isNonStrictVerifySslCertificatesOfServices()
```
condition be added also here and in the other conditions?

Reply:
We can, but in my opinion it shouldn't happen that someone configures the api ml to not validate certificates but be strict, the second condition does not really matter in this stage

Reply:
ok sounds good, thanks for the clarification